website/docs/r/storage_account_network_rules.html.markdown

```diff
@@ -88,6 +88,17 @@ The following arguments are supported:
 
 -> **NOTE** User has to explicitly set `virtual_network_subnet_ids` to empty slice (`[]`) to remove it.
 
+* `resource_access_rules` - (Optional) A `resource_access_rules` block as defined below.
+
+---
+
+A `resource_access_rules` block supports the following:
+
+* `resource_id` - (Required) The resource id of the `azurerm_private_endpoint` of the resource access rule.
+
+* `tenant_id` - (Optional) The tenant id of the `azurerm_private_endpoint` of the resource access rule. Defaults to the current tenant id.
```

Comment:
if this is just for private link
```suggestion
* `private_endpoint_id` - (Required) The resource id of the `azurerm_private_endpoint` of the resource access rule.

* `private_endpoint_tenant_id` - (Optional) The tenant id of the `azurerm_private_endpoint` of the resource access rule. Defaults to the current tenant id.
```

and maybe pulling these out to the top level?

Reply:
Since there could be several private_endpoint to be connected, shall I keep the block and change the block name?

Reply:
I've update the acctests to connect two private endpoints.